### PR TITLE
Fix polyform libs names

### PR DESCRIPTION
**Pull request by paluh**

### CODE REVIEW

File: new-packages.json [hunk 1]
```diff
@@ -134,9 +134,9 @@
   "purescript-halogen-store": "https://github.com/thomashoneyman/purescript-halogen-store.git",
   "purescript-elasticsearch": "https://github.com/ConnorDillon/purescript-elasticsearch.git",
   "purescript-read-generic": "https://github.com/Swordlash/purescript-read-generic.git",
-  "purescript-batteries-core": "https://github.com/purescript-polyform/batteries-core.git",
-  "purescript-batteries-env": "https://github.com/purescript-polyform/batteries-env.git",
-  "purescript-batteries-json": "https://github.com/purescript-polyform/batteries-json.git",
-  "purescript-batteries-urlencoded": "https://github.com/purescript-polyform/batteries-urlencoded.git",
+  "purescript-polyform-batteries-core": "https://github.com/purescript-polyform/batteries-core.git",
```
Comment (f-f) [inline new-packages.json]: The other packages don't have a `bower.json` so they can be renamed - speaking of which, could you add a `bower.json` to them once we merge this change? - but `batteries-core` has one, so we should leave the existing entry in and add a new one instead.

```suggestion
  "purescript-batteries-core": "https://github.com/purescript-polyform/batteries-core.git",
  "purescript-polyform-batteries-core": "https://github.com/purescript-polyform/batteries-core.git",
```

**Review by f-f — CHANGES_REQUESTED** [review]

**Comment by f-f** [comment]
@paluh any update on this?

**Comment by paluh** [comment]
@f-f sorry about this. I'm going to clean it up during the weekend.

**Comment by f-f** [comment]
@paluh no worries! We're about to import all packages in the new Registry and it's going to be much harder to make such changes once we do that. So I wanted to make sure that we still need this change, and if that's the case that we merge it asap

**Pull request closed without merging**